renter/download.go

```diff
@@ -23,11 +23,14 @@ import (
 	"github.com/klauspost/reedsolomon"
 )
 
-func (r *Renter) Download(fileId string, destPath string) error {
+func (r *Renter) Download(fileId string, destPath string, versionNum *int) error {
```

Comment:
Is that... a pointer to an int? I'm curious as to why you're using that.

Reply:
Weak type system: This is the most straightforward way to make versionNum optional (in both the requestJSON and Download function).

Reply:
Ah, gotcha.